src/controllers/shop/checkout-item.controller.js

```diff
@@ -45,26 +54,28 @@ const checkoutItem = async (req, res) => {
       product.stock -= qty;
       await product.save();
 
-      const cart = await Cart.findOne({
-        where: { user_id: user.id, product_id },
-      });
-      if (!cart) {
-        return response(res, 404, false, "Cart not found", null);
-      }
+      // const cart = await Cart.findOne({
+      //   where: { user_id: user.id, product_id },
+      // });
+
+      // if (!cart) {
+      //   return response(res, 404, false, "Cart not found", null);
+      // }
```

Comment:
I've removed this

Reply:
refer to commit : https://github.com/Belega-Village-UNUD/backend-api-edb/pull/51/commits/63b0a8f98a2e368a94cc9eef20db18b9d6fc6ac6